src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickedEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class DatePickedEventArgs
+	{
+		public DatePickedEventArgs(DateTimeOffset newDate, DateTimeOffset oldDate)
```

Comment:
For UWP compatibility this should be internal, and you should put a public parameterless ctor (even though it's useless)

Reply:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.datepickedeventargs.-ctor#Windows_UI_Xaml_Controls_DatePickedEventArgs__ctor